integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/InsertIntoDataSourceVisitor.java

```diff
@@ -48,6 +48,7 @@ public InsertIntoDataSourceVisitor(OpenLineageContext context) {
                         .newDatasetFacets(
                             ds.getFacets().getDocumentation(),
                             ds.getFacets().getDataSource(),
+                            ds.getFacets().getVersion(),
```

Comment:
To get the dataset version, the method calls would look like:

```java
dataset.getFacets().getVersion().getDatasetVersion()
```

which, is a bit awkward. Another option would be to reverse the method calls (i.e. naming):

```
dataset.getFacets().getDatasetVersion().getVersion()
```

In the future, we might have a job version facet that can follow the same naming strategy:

```
job.getFacets().getJobVersion().getVersion()
```